readme.md

```diff
@@ -117,7 +117,7 @@ import { rateLimit } from 'express-rate-limit'
 const limiter = rateLimit({
 	windowMs: 15 * 60 * 1000, // 15 minutes
 	max: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes)
-	standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
+	standardHeaders: 'draft-7', // Set `RateLimit` and `RateLimit-Policy`` headers
 	legacyHeaders: false, // Disable the `X-RateLimit-*` headers
```

Comment:
I might rework these comments so that they're accurate even if the value is changed.